db/migrate/20150312144232_migrate_websockets_setting.rb

```diff
@@ -0,0 +1,19 @@
+class MigrateWebsocketsSetting < ActiveRecord::Migration
+  def up
+    return unless encrypt = Setting.find_by_name("websockets_encrypt")
+    encrypt.settings_type = "boolean"
+    if encrypt.value == "auto"
+      encrypt.value = (Setting.find_by_name("websockets_ssl_key").nil? ||
```

Comment:
Check the `.value` of the settings, or use `Setting[:websockets_ssl_key]` to see the combined (default+value) value.  It's currently checking if a Setting object is nil or not.